pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -105,6 +105,22 @@ func (a *Assignment) ToAPI() []kueue.PodSetAssignment {
 	return psFlavors
 }
 
+func (a *Assignment) TotalRequestsFor(wl *workload.Info) cache.FlavorResourceQuantities {
```

Comment:
nit: This looks fairly similar to `workload.FlavorResourceUsage`. In particular the output seems the same.

Is it feasible to commonize it?